modules/fundamental/src/purl/service/mod.rs

```diff
@@ -317,6 +353,61 @@ impl PurlService {
         })
     }
 
+    fn build_cyclonedx_license_query(license_query: Query) -> Result<SelectStatement, Error> {
+        Ok(Self::create_license_filtering_base_query()
+            .filtering_with(
+                license_query,
+                license::Entity
+                    .columns()
+                    .translator(|field, operator, value| match field {
+                        "license" => Some(format!("text{operator}{value}")),
+                        _ => None,
+                    }),
+            )?
+            .into_query())
+    }
+
+    fn build_spdx_license_query(license_query: Query) -> Result<SelectStatement, Error> {
+        Ok(Self::create_license_filtering_base_query()
+            .filtering_with(
+                license_query,
+                Columns::default()
+                    .add_expr(
+                        "expanded_license",
+                        SimpleExpr::FunctionCall(
+                            Func::cust("expand_license_expression".into_identity())
```

Comment:
Bonus points for creating an `Ident` for this. Like: https://github.com/guacsec/trustify/blob/986f992cdf67c0504ccd52f1a0cd7d06f95cad14/common/src/db/func.rs#L51-L58